dev/util.h

```diff
@@ -13,7 +56,7 @@ namespace sqlite_orm {
             }
         }
 
-        static void perform_void_exec(sqlite3* db, const std::string& query) {
+        inline void perform_void_exec(sqlite3* db, const std::string& query) {
```

Comment:
why did you change `static` to `inline`? `static` was placed to make this function file private

Reply:
I just aligned it to all other functions in sqlite_orm. `perform_step()` in the very same file is inline, as are all other free-standing functions.
Also because it's in the internal namespace there's no need to make it linker private?

So, counter question: why would you not use `inline`?

Reply:
we can use `inline` it is ok. The question is about `static`. It is important to make this function file private to eliminate opportunities to run string queries for users. Private `internal` namespace is not private actually cause C++ doesn't have private namespaces. And C++ doesn't have private non-class functions. That is why this function is `static`

Reply:
Oh I understand your point. So you want to make it private in regard to linking as well and prohibit the compiler from accidentally creating a full definition that the linker exports? Or otherwise what's your notion of "private", or "users"? Even a static function isn't private in C++ when not defined in a .cpp file.

Reply:
1) "private" is the same as "private" in C#, Swift or any other popular languages.
2) "users" = developers who uses `sqlite_orm`
3) if static function in '.h' file is accessible then ignore my words. Then I forgot about it